_dashboards/branding.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: Custom branding
+nav_order: 200
+---
+
+# OpenSearch Dashboards custom branding
+Introduced 1.2
+{: .label .label-purple }
+
+By default, OpenSearch Dashboards uses the OpenSearch logo, but if you want to use custom branding elements such as the favicon or main Dashboards logo, you can do so by editing `opensearch_dashboards.yml` or by including a custom `opensearch_dashboards.yml` file when you start your OpenSearch cluster.
+
+For example, if you're using Docker to start your OpenSearch cluster, include the following lines in the `opensearch-dashboards` section of your `docker-compose.yml` file:
+
+```
+volumes:
+  - ./opensearch_dashboards.yml:/usr/share/opensearch-dashboards/config/opensearch_dashboards.yml
+```
+
+Doing so replaces the Docker image's default `opensearch_dashboards.yml` with your custom `opensearch_dashboards.yml` file, so be sure to include your desired as well. For example, if you want to configure TLS for OpenSearch Dashboards, see [Configure TLS for OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/install/tls).
```

Comment:
Not sure I understand "to include your desired as well." Does this mean use your desired file name?

Reply:
That should actually say "your desired settings." Thanks for catching!